### PR TITLE
Removed outdated configurtion files and package, ...

### PR DESCRIPTION
**Pull request by joergsteffens**
### Changed
- Package bareos-database-postgresql: add recommendation for package dbconfig-pgsql.
- Adapt the init scripts for some platform to not refer to an specific (outdated) configuration file, but use the default config file instead.

### Removed
 - Removed outdated configuration files (example files).
- Removed package **bareos-devel**.
- Removed package **bareos-regress** and **bareos-regress-config**. The package **bareos-regress** has not been build for a long time.
- Removed unused script **breload**.
- Removed some workaround for Ubuntu 8.04.
- Removed outdated support for platforms Alpha, BSDi, Gentoo, Irix and Mandrake.
- Removed language support files for the core daemons, as these files are outdated and not used at all.
- Removed package lists for platforms no longer built: Fedora_30.x86_64, RHEL_6.x86_64, SLE_12_SP4.x86_64, openSUSE_Leap_15.0.x86_64, openSUSE_Leap_15.1.x86_64.

### CODE REVIEW

**Comment by joergsteffens** [comment]
While I assume that the content of this PR is correct, I plan to rearranged the commits (after a review).

File: CHANGELOG.md [hunk 1]
```diff
@@ -12,10 +12,21 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
 - added reload commands to systemd service [PR #694]
 
 ### Changed
+- Package bareos-database-postgresql: add recommendation for package dbconfig-pgsql.
+- Adapt the init scripts for some platform to not refer to an specific (outdated) configuration file, but use the default config file instead.
```
Comment (pstorz) [inline CHANGELOG.md:16]:
```suggestion
- Adapt the init scripts for some platform to not refer to a specific (outdated) configuration file, but to use the default config file instead.
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -12,10 +12,21 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
...
 
 ### Deprecated
 
 ### Removed
+- Removed outdated configuration files (example files).
+- Removed package bareos-devel.
+- Removed package bareos-regress and bareos-regress-config. The package bareos-regress have not been build for a long time.
```
Comment (pstorz) [inline CHANGELOG.md:23]:
```suggestion
- Removed package bareos-regress and bareos-regress-config. The package bareos-regress has not been build for a long time.
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -12,10 +12,21 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
...
 
 ### Changed
+- Package bareos-database-postgresql: add recommendation for package dbconfig-pgsql.
+- Adapt the init scripts for some platform to not refer to an specific (outdated) configuration file, but use the default config file instead.
 
 ### Deprecated
 
 ### Removed
+- Removed outdated configuration files (example files).
+- Removed package bareos-devel.
```
Comment (pstorz) [inline CHANGELOG.md:22]:
```suggestion
- Removed package "bareos-devel".
```

Comment (pstorz) [inline CHANGELOG.md:22]:
```suggestion
- Removed package "bareos-devel".
```

File: CHANGELOG.md [hunk 4]
```diff
@@ -12,10 +12,21 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
...
+- Removed package bareos-regress and bareos-regress-config. The package bareos-regress have not been build for a long time.
+- Removed unused script breload.
```
Comment (pstorz) [inline CHANGELOG.md:24]:
```suggestion
- Removed unused script "breload".
```

File: CHANGELOG.md [hunk 5]
```diff
@@ -12,10 +12,21 @@ and since Bareos version 20 this project adheres to [Semantic Versioning](https:
...
+- Removed some workaround for Ubuntu 8.04.
+- Removed outdated support for platfrom Mandrake.
+- Removed outdated support for platfrom Gentoo.
+- Removed language support files for the core daemons, as these files are outdated and not used at all.
+- Removed package lists for platform no longer build: Fedora_30.x86_64, RHEL_6.x86_64, SLE_12_SP4.x86_64, openSUSE_Leap_15.0.x86_64, openSUSE_Leap_15.1.x86_64.
```
Comment (pstorz) [inline CHANGELOG.md:29]:
```suggestion
- Removed package lists for platforms no longer built: Fedora_30.x86_64, RHEL_6.x86_64, SLE_12_SP4.x86_64, openSUSE_Leap_15.0.x86_64, openSUSE_Leap_15.1.x86_64.
```

File: core/platforms/bsdi/bareos-dir.in [hunk 6]
```diff
@@ -12,7 +12,7 @@ RETVAL=0
 case "$1" in
     start)
        echo "Starting the Bareos Director: "
-       @sbindir@/bareos-dir $2 -c @confdir@/bareos-dir.conf
+       @sbindir@/bareos-dir $2
```
Comment (pstorz) [inline core/platforms/bsdi/bareos-dir.in:15]: If bsdi means this: 
https://en.wikipedia.org/wiki/Berkeley_Software_Design
then we should remove the complete "bsdi" directory I think.

File: core/platforms/irix/bareos-dir.in [hunk 7]
```diff
@@ -12,7 +12,7 @@ RETVAL=0
 case "$1" in
     start)
        echo "Starting the Bareos Director: "
-       @sbindir@/bareos-dir $2 -c @confdir@/bareos-dir.conf
+       @sbindir@/bareos-dir $2
```
Comment (pstorz) [inline core/platforms/irix/bareos-dir.in:15]: I think that IRIX also can be considered dead so we should remove it completely.

File: core/scripts/bareos-regress.conf.in [hunk 8]
```diff
@@ -1,3 +1,10 @@
+#
```
Comment (pstorz) [inline core/scripts/bareos-regress.conf.in:1]: Does the regression testing with this config work? If the regression tests do not work anymore we could also remove everything that is left from them also.

File: regress/README [hunk 9]
```diff
@@ -11,50 +11,26 @@ with your production Catalog because the tables will all
 be deleted (dropped and recreated).
```
Comment (pstorz) [inline regress/README:11]: We should refer to the systemtests in ctest and say that the regression tests are at deprecated.

**Review by pstorz — COMMENTED** [review]
I

File: CHANGELOG.md (hunk 3 above)
Comment (joergsteffens) [reply CHANGELOG.md]: Modified to bold (**).

File: CHANGELOG.md (hunk 4 above)
Comment (joergsteffens) [reply CHANGELOG.md]: Modified to bold (**).

File: core/platforms/bsdi/bareos-dir.in (hunk 6 above)
Comment (joergsteffens) [reply core/platforms/bsdi/bareos-dir.in]: Removed.

File: core/platforms/irix/bareos-dir.in (hunk 7 above)
Comment (joergsteffens) [reply core/platforms/irix/bareos-dir.in]: Removed.

File: regress/README (hunk 9 above)
Comment (joergsteffens) [reply regress/README:18]: Added warning.

File: core/scripts/bareos-regress.conf.in (hunk 8 above)
Comment (joergsteffens) [reply core/scripts/bareos-regress.conf.in:1]: I think, this file is still useful. I recommend to handle a regress specific cleanup in a separate PR.

**Comment by joergsteffens** [comment]
Rebased commits. I consider this PR ready for merge. Additional changes can be handled in additional PRs.

**Review by pstorz — APPROVED** [review]
Looks good to  me.

**Pull request merged**